pipeline/src/test_util.rs

```diff
@@ -74,13 +73,14 @@ pub fn regular_test(file_name: &str, inputs: &[i32]) {
 pub fn regular_test_without_small_field(file_name: &str, inputs: &[i32]) {
     let inputs_gl = inputs.iter().map(|x| GoldilocksField::from(*x)).collect();
     let pipeline_gl = make_prepared_pipeline(file_name, inputs_gl, vec![]);
+
     test_mock_backend(pipeline_gl.clone());
-    test_pilcom(pipeline_gl.clone());
-    gen_estark_proof(pipeline_gl);
+    run_pilcom_with_backend_variant(pipeline_gl.clone(), BackendVariant::Composite).unwrap();
```

Comment:
I thought pilcom only works with Monolithic?

Reply:
My bad, I meant the non-composite pilcom backend. The non-composite plonky3 backend does support vadcop for example. The composite pilcom backend is of limited value since the links get removed when splitting the pil.